features/networking/multicast.feature

```diff
@@ -13,18 +13,28 @@ Feature: testing multicast scenarios
   @heterogeneous @arm64 @amd64
   Scenario: OCP-12926 pods should be able to subscribe send and receive multicast traffic
     # create some multicast testing pods
+    Given I store the schedulable workers in the :nodes clipboard
     Given I have a project
     And evaluation of `project.name` is stored in the :proj1 clipboard
     Given I obtain test data file "networking/multicast-rc.json"
-    When I run the :create client command with:
-      | f | multicast-rc.json |
+    When I run oc create over "multicast-rc.json" replacing paths:
+      | ["spec"]["replicas"] | 2                       |
```

Comment:
@huiran0826 If this step only create two pods  which are in the different nodes, do we still need a case for the pods are in the same node? Other case may covered. To make things simple, keep OCP-12926 as test pods in the same node, and using OCP-45022 to test Multicast traffic cross nodes.  WDYT?

Reply:
@weliang1  Actually  OCP-12926 will cover both situation, same nodes and different nodes. 
Currently we only have this OCP-45022 for multicast as manual left, now I prefer to just update this case than to create a new case in go which might need a lot of effort and research as we don't have the basic utilizes for this kind of cases.

Reply:
OK. 
/lgtm